Packaging.props

```diff
@@ -20,7 +20,7 @@
     <PlatformPackageVersion Condition="'$(PlatformPackageVersion)' == ''">1.2.0</PlatformPackageVersion>
 
     <!-- by default all packages will use the same version which revs with respect to product version -->
-    <PackageVersion Condition="'$(PackageVersion)' == ''">4.4.1</PackageVersion>
+    <PackageVersion Condition="'$(PackageVersion)' == ''">4.4.2</PackageVersion>
```

Comment:
Just realized we are building 4.4.1 stable from this branch all the time since last release. Once this PR is merge and we get a good 4.4.2 build, please remember to update this to 4.4.3.

Reply:
Also, package publishing has been disabled in 2.0.0, I did one build last night to make sure publishing works so now we have a stable 4.1.1 package on MyGet.

I won't build again until this PR goes in.